src/app/main/cache/cache.service.ts

```diff
@@ -35,9 +35,6 @@ export class CacheService {
             // console.log(`%c Cache returns typeof:` + (typeof of(this.cache.get(key).value)), 'color: green');
 
             // returns observable
-            if (key === 'kilchenmann') {
```

Comment:
hahaha.. I haven't checked the code. The console.logs were only visible in my profile. I did not remember that. But thanks for removing it.